api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/PaymentCycleNotificationHandler.java

```diff
@@ -54,11 +58,22 @@ private boolean voucherEntitlementIndicatesNewChildFromPregnancy(PaymentCycle pa
 
     private void handleUpcomingBirthdayEmails(PaymentCycle paymentCycle) {
         NextPaymentCycleSummary nextPaymentCycleSummary = childDateOfBirthCalculator.getNextPaymentCycleSummary(paymentCycle);
-        if (nextPaymentCycleSummary.hasChildrenTurningFour()) {
-            upcomingBirthdayEmailHandler.sendChildTurnsFourEmail(paymentCycle, nextPaymentCycleSummary);
-        }
-        if (nextPaymentCycleSummary.hasChildrenTurningOne()) {
-            upcomingBirthdayEmailHandler.sendChildTurnsOneEmail(paymentCycle, nextPaymentCycleSummary);
+        boolean isNotPregnant = isNotPregnant(paymentCycle);
```

Comment:
I don't think this adds anything making this a local variable as its not reused and looks nearly identical, we should inline this.

Reply:
true - it's a refactoring artefact I thought we'd removed again, oops.

Reply:
Done